api/v1/cluster_types.go

```diff
@@ -188,6 +199,24 @@ type OnlineConfiguration struct {
 	ImmediateCheckpoint *bool `json:"immediateCheckpoint,omitempty"`
 }
 
+// VolumeSnapshotRetryConfiguration contains parameters to control retry behavior
+// for operations that may fail transiently, such as volume snapshot creation
+type VolumeSnapshotRetryConfiguration struct {
+	// The maximum time to wait for a snapshot operation to complete before
+	// considering it failed. Default is 10 minutes.
+	// +optional
+	// +kubebuilder:default:="10m"
+	// +kubebuilder:validation:Pattern=^([0-9]+(m|s))+$
```

Comment:
We should probably support hour here and return a helpful message when validation fails.